styles/index.css

```diff
@@ -0,0 +1,7 @@
+img{
+    width:400px;
+    height:300px;
+    background-size:initial;
+    background-repeat: no-repeat;
+   
+}
```

Comment:
I would suggest using a flex layout or grid layout so the user doesn't have to scroll to see the emojis when the the image changes. Here is a great place to start https://css-tricks.com/snippets/css/a-guide-to-flexbox/